docs/articles/authn-authz.md

```diff
@@ -364,7 +368,7 @@ Like zot, dex uses a configuration file for setup. To specify zot as a client in
 staticClients:
   - id: zot-client
     redirectURIs:
-      - 'http://zot-example.com:8080/auth/callback/dex'
+      - 'http://zot.example.com:8080/auth/callback/dex'
```

Comment:
While dex as a service zot connects to continues to be supported, the zot endpoints and configuration options have been recently renamed, in order to be more generic, in: https://github.com/project-zot/zot/pull/1691/files

So we do support a dex server, but the user can also use also other oidc servers.
See the PR I linked for more details. basically the zot config key was renamed from `dex` to `oidc`, and this endpoint used `oidc` instead of `dex`